requirements.d/test.txt

```diff
@@ -6,7 +6,7 @@ mock
 pre-commit>=0.3.6
 pylint
 # FIXME: fails with `assert (<DoctestModule 'tests/spec/cli.py'> is None or <DoctestModule 'tests/spec/cli.py'> in [<Session 'pgctl'> ...`
-pytest!=2.8.0,!=2.8.1
+pytest!=2.8.0,!=2.8.1,!=2.8.2
```

Comment:
looks like it may be fixed in 2.8.3

https://github.com/pytest-dev/pytest/issues/1057